async-jms/apps/async-jms/src/main/java/com/redhat/training/model/Order.java

```diff
@@ -0,0 +1,127 @@
+package com.redhat.training.model;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+public class Order implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+	
+	@JsonProperty("ID")
+	private Integer id;
+	@JsonIgnore
+	private Date orderDate = new Date();
+	@JsonProperty("Discount")
+	private BigDecimal discount;
+	@JsonProperty("Delivered")
+	private Boolean delivered=false;
+	@JsonProperty("Desc")
+	private String description;
+
+	public Integer getId() {
+		return id;
+	}
+
+	public void setId(Integer i) {
+        	this.id = i;
+	}
+
+	public Date getOrderDate() {
+		return orderDate;
+	}
+
+	public void setOrderDate(Date orderDate) {
+		this.orderDate = orderDate;
+	}
+
+	public BigDecimal getDiscount() {
+		return discount;
+	}
+
+	public void setDiscount(BigDecimal discount) {
+		this.discount = discount;
+	}
+
+	public Boolean getDelivered() {
+		return delivered;
+	}
+
+	public void setDelivered(Boolean delivered) {
+		this.delivered = delivered;
+	}
+
+	public void deliver() {
+		setDelivered(true);
+	}
+
+	public void setDescription(String desc) {
+		this.description = desc;
+	}
+
+	public String getDescription() {
+		return this.description;
+	}
+
+	@Override
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result + ((delivered == null) ? 0 : delivered.hashCode());
+		result = prime * result + ((discount == null) ? 0 : discount.hashCode());
+		result = prime * result + ((id == null) ? 0 : id.hashCode());
+		result = prime * result + ((orderDate == null) ? 0 : orderDate.hashCode());
+		return result;
```

Comment:
Empty lines before return statements, specially if there is preceded by a big block of code.